guides/common/modules/proc_creating-rhui-alternate-content-sources-by-using-cli.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="creating-rhui-alternate-content-sources-by-using-cli"]
+= Creating RHUI alternate content sources by using Hammer CLI
+
+[role="_abstract"]
+You can use RHUI alternate content sources to configure your {SmartProxy} to sync content directly from Red{nbsp}Hat CDN.
+
+.Prerequisites
+* Generate the client entitlement certificates for the required repos on the RHUA node as described in {RHDocsBaseURL}red_hat_update_infrastructure/4/html/configuring_and_managing_red_hat_update_infrastructure/assembly_cmg-creating-client-ent-cert-config-rpm_configuring-and-managing-red-hat-update-infrastructure#proc_cmg-creating-client-entitlement-certificate_assembly_cmg-creating-client-ent-cert-config-rpm[Creating a client entitlement certificate with the Red Hat Update Infrastructure Management Tool] in _Configuring and Managing Red Hat Update Infrastructure_.
+* Import the client entitlement certificates into {Project}.
+For more information, see {ContentManagementDocURL}Importing_Custom_SSL_Certificates_content-management[Importing {customssl} certificates] in _{ContentManagementDocTitle}_.
+* Obtain a list of the subpaths for the required repositories.
+Execute the following command on your RHUA server:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# rhui-manager repo info --repo_id _My_Repo_ID_
+----
+* Note that the alternate content source paths consist of a base URL appended with the subpaths that you provide.
+For example, if your base URL is `\https://{server-example-com}` and your subpaths are `rhel7/` and `rhel8/`, then both `\https://{server-example-com}/rhel7/` and `\https://{server-example-com}/rhel8/` will be searched.
+
+.Procedure
+. Create a RHUI alternate content source:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+$ hammer alternate-content-source create \
+--alternate-content-source-type rhui \
+--base-url "_https://rhui-cds-node.example.com/pulp/content_" \
+--name "_My_ACS_Name_" \
+--smart-proxy-ids __My_{smart-proxy-context-titlecase}_ID_1__,__My_{smart-proxy-context-titlecase}_ID_2__ \
+--ssl-client-cert-id _My_SSL_Client_Certificate_ID_ \
+--ssl-client-key-id _My_SSL_Client_Key_ID_ \
+--subpaths _path/to/repo/1/,path/to/repo/2/_ \
+--verify-ssl _true_
+----
+. Refresh the alternate content source:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+$ hammer alternate-content-source refresh --name _My_Alternate_Content_Source_
```

Comment:
```suggestion
$ hammer alternate-content-source refresh --name _My_ACS_Name_
```
The example doesn't match the ACS name as in the previous command.
Alternatively, you can change it above.